examples/riddle-prep-buddy-v2/README.md

```diff
@@ -40,9 +40,7 @@ A small tkinter application students can use to practice for the riddles section
 
 1. **Run the Server Notebook:**
 
-    - Make a copy of [this notebook](https://colab.research.google.com/drive/1-1s5k-mjOhEuuMJBDbd_ei5KRxrgOOKS?usp=sharing) in Google Colab
-    - Follow the instructions to start the server
-    - Copy the output `ngrok` URL.
+    Open the `[Release] Riddle Prep App Server.ipynb` file, preferably in Colab if you don't have a GPU on your, follow the instructions to start the server, and copy the `ngrok` URL.
```

Comment:
typo here `on your ..`? Is the notebook view only? Also the readme change here no longer has a link to the notebook. If that is a bug please re-add it